pkg/controllers/util/k8s.go

```diff
@@ -0,0 +1,40 @@
+/*
+ (c) Copyright [2021-2023] Open Text.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package util
```

Comment:
util isn't the best name for a package. It's actually discouraged in the Go style guide (https://google.github.io/styleguide/go/best-practices.html#util-packages). Could we rename the package to vk8s? It can be a package for k8s helpers used by the vertica operator. It should move to be a directory directly under pkg/ too.

Reply:
Yes I like it.